.pipelines/templates/PackageBuild.yml

```
@@ -194,7 +199,8 @@ steps:
         $delta_fetch_arg \
         $quick_rebuild_packages_arg \
         $run_check_arg \
-        $use_ccache_arg
+        $use_ccache_arg \
+        $toolchain_archive_arg
```

Nit-pick request to keep these in alphabetical order.

```suggestion
        $toolchain_archive_arg \
        $use_ccache_arg

```